geo/src/algorithm/minimum_rotated_rect.rs

```diff
@@ -75,11 +123,11 @@ mod test {
         assert_eq!(
             mbr.exterior(),
             &LineString::from(vec![
-                (1.7000000000000006, 24.6),
-                (1.4501458363715918, 30.446587428904767),
-                (14.4, 31.0),
-                (14.649854163628408, 25.153412571095235),
-                (1.7000000000000006, 24.6),
+                (1.7000000000000004, 24.600000000000005),
+                (14.64985416362841, 25.153412571095235),
```

Comment:
Maybe you noticed this but it looks like the `y` value for the 2nd and 4th point have been swapped. 😕

Reply:
The points have been swapped. I think the vertices are in the reverse order.

Reply:
Oh right, I stared at it too long and confused myself. 😄 

Is the new ordering correct or is that something that needs to be fixed yet?

Reply:
> Is the new ordering correct or is that something that needs to be fixed yet?

It's the same rectangle in the opposite winding order (P4 P3 P2 P1 P4 instead of P1 P2 P3 P4 P1). I can probably match the order, but only up to the starting vertex, e.g. P2 P3 P4 P1 P2.